content/user/ui/overview/Basic-quick-start-guide.md

```diff
@@ -0,0 +1,111 @@
+<!--
+title: "Quick Start Guide"
+description: "Overview of Contrast UI for new users"
+tags: "TeamServer user applications vulnerabilities"
```

Comment:
Did you revise the tags here? We probably want to do that to better match this article. Things like "onboarding" "getting started" "new user", etc. Thanks!